File: Com.Fubei.Api.OpenApi/Biz/AgentApi/Model/Response/AOrderQueryResultEntity.cs
using System;
using System.Collections.Generic;
using System.Text;
using Com.Fubei.Api.OpenApi.Biz.AgentApi.Model.Request;
using Com.Illuminati.Galileo.Model;
using Newtonsoft.Json;

namespace Com.Fubei.Api.OpenApi.Biz.AgentApi.Model.Response
{
    public class AOrderQueryResultEntity : BaseEntity
    {
        [JsonProperty("merchant_order_sn")]
        public string MerchantOrderSn { get; set; }

        [JsonProperty("order_sn")]
        public string OrderSn { get; set; }

        [JsonProperty("ins_order_sn")]
        public string InsOrderSn { get; set; }

        [JsonProperty("channel_order_sn")]
        public string ChannelOrderSn { get; set; }

        [JsonProperty("order_status")]
        public string OrderStatus { get; set; }

        [JsonProperty("pay_type")]
        public string PayType { get; set; }

        [JsonProperty("total_amount")]
        public decimal TotalAmount { get; set; }

        [JsonProperty("net_amount")]
        public decimal? NetAmount { get; set; }

        [JsonProperty("buyer_pay_amount")]
        public decimal? BuyerPayAmount { get; set; }

        [JsonProperty("fee")]
        public decimal? Fee { get; set; }

        [JsonProperty("store_id")]
        public int StoreId { get; set; }

        [JsonProperty("cashier_id")]
        public int CashierId { get; set; }

        [JsonProperty("user_id")]
        public string UserId { get; set; }

        [JsonProperty("finish_time")]
        public string FinishTime { get; set; }

        [JsonProperty("device_no")]
        public string DeviceNo { get; set; }

        [JsonProperty("attach")]
        public string Attach { get; set; }

        [JsonProperty("payment_list")]
        public List<APaymentItemEntity> PaymentList { get; set; }

        [JsonProperty("alipay_extend_params")]
        public AlipayExtendParamsEntity AlipayExtendParams { get; set; }
    }
}

File: Com.Fubei.OpenApi.Vb/Biz/AgentApi/Model/Response/AOrderQueryResultEntity.vb
Imports Com.Illuminati.Galileo.Model
Imports Newtonsoft.Json

Namespace Biz.V2.Model.Response
    
    ' 订单查询请求响应实体
    Public Class AOrderQueryResultEntity
        Inherits BaseEntity

        <JsonProperty("merchant_order_sn")>
        Public Property MerchantOrderSn As String

        <JsonProperty("order_sn")>
        Public Property OrderSn As String

        <JsonProperty("ins_order_sn")>
        Public Property InsOrderSn As String

        <JsonProperty("channel_order_sn")>
        Public Property ChannelOrderSn As String

        <JsonProperty("order_status")>
        Public Property OrderStatus As String

        <JsonProperty("pay_type")>
        Public Property PayType As String

        <JsonProperty("total_amount")>
        Public Property TotalAmount As Decimal

        <JsonProperty("net_amount")>
        Public Property NetAmount As Decimal?

        <JsonProperty("buyer_pay_amount")>
        Public Property BuyerPayAmount As Decimal?

        <JsonProperty("fee")>
        Public Property Fee As Decimal?

        <JsonProperty("store_id")>
        Public Property StoreId As Integer

        <JsonProperty("cashier_id")>
        Public Property CashierId As Integer

        <JsonProperty("user_id")>
        Public Property UserId As String

        <JsonProperty("finish_time")>
        Public Property FinishTime As String

        <JsonProperty("device_no")>
        Public Property DeviceNo As String

        <JsonProperty("attach")>
        Public Property Attach As String

        <JsonProperty("payment_list")>
        Public Property PaymentList As List(Of APaymentItemEntity)

        <JsonProperty("alipay_extend_params")>
        Public Property AlipayExtendParams As AlipayExtendParamsEntity
    End Class
End Namespace
